OPAL/si/src/main/scala/org/opalj/fpcf/scheduling/MaximumPhaseScheduling.scala

```diff
@@ -98,7 +97,39 @@ abstract class MultiplePhaseScheduling extends SchedulingStrategy {
             computePhase(ps, phaseAnalyses, remainingAnalyses)
         }
 
-        schedule
+        if(config.getBoolean(DisableCleanup)) calculateDeletions(schedule, ps, config) else schedule
+    }
+
+    private def calculateDeletions[A](schedule: List[PhaseConfiguration[A]], ps: PropertyStore, config: Config): List[PhaseConfiguration[A]] = {
+        def getSetOrNilForProperty(propertyKey: String) = {
+            if (!config.hasPath(propertyKey)) Nil
+            else {
+                val rawValue = config.getString(propertyKey).trim
+                if (rawValue.isEmpty) Set.empty[Int]
+                else rawValue.split(",").map(_.trim.toInt).toSet
```

Comment:
We can't specify property keys by their ids, those are not guaranteed to be stable. If you need them specified, you must do so by name. But I still suggest a programmatical way to do it instead of (ab)using the config.